src/dmd/tokens.d

```diff
@@ -693,6 +694,7 @@ extern (C++) struct Token
         TOK.interval: "interval",
         TOK.voidExpression: "voidexp",
         TOK.cantExpression: "cantexp",
+        TOK.errExtraInfo : "errExtraInfo",
```

Comment:
(4) ...and then you shouldn't need this anymore.

Reply:
If that is not feasible, what about just adding a new `__gshared CTFEExp` here?  https://github.com/dlang/dmd/blob/ed6386ada4549d70797c3d09e51e05756de174d4/src/dmd/ctfeexpr.d#L233-L237

Reply:
This seems like the right way to go. Thanks!

Reply:
Done.

Reply:
Consider a different name, but like `showCtfeContext`, or something like that.  Though I'd be willing to merge this without a name change.